hydroshare/settings.py

```diff
@@ -529,5 +529,10 @@
             'level': 'DEBUG',
             'propagate': False,
         },
+        'ref_ts': {
+            'handlers':['djangolog'],
+            'propagate': False,
+            'level':'DEBUG',
+        },
```

Comment:
@zhiyuli There needs to be a good reason to define a new logger for a given resource type.  In my own development, I haven't seen a reason not to use the 'django' logger.